model/schema/modCorpusAssociation.yaml

```diff
@@ -0,0 +1,87 @@
+id: https://github.com/alliance-genome/agr_curation_schema/src/schema/modCorpusAssociation
+
+imports:
+  - core
+  - mod
+  - reference
+  - linkml:types
+  - informationContentEntity
+
+prefixes:
+  alliance: 'http://alliancegenome.org'
+  linkml: 'https://w3id.org/linkml/'
+  gff: 'https://w3id.org/gff'
+  faldo: 'http://biohackathon.org/resource/faldo#'
+  biolink: 'https://w3id.org/biolink/vocab/'
+  NLMID: 'https://www.ncbi.nlm.nih.gov/nlmcatalog/?term='
+  schema: 'http://schema.org/'
+
+classes:
+  ModCorpusAssociation:
+    is_a: InformationContentEntity
+    description: >-
+      For a given reference and Mod, whether it is inside corpus,
+      outside corpus, or needs review, as well as where this
+      sorting came from.
+    notes: >-
+      There should be only one corpus call for any given reference-mod pair.
+    slots:
+      - corpus
+      - date_created
+      - date_updated
+      - mod_corpus_association_id
+      - mod_corpus_sort_source
+      - mod_id
+      - reference_id
```

Comment:
Is there a reason for the slot to just point to an ID and not the mod/reference object itself?